string cam_id
---
string pickled_cmds
